label_check/entrypoint.sh

```diff
@@ -92,7 +92,7 @@ main(){
 
     echo "...done"
 
-    message="*LABEL CHECK*: Attention!! New deploy action launched."
+    message="*LABEL CHECK*: *${label_to_check}*: Attention!! New deploy action launched."
```

Comment:
This is good, but you have to set this variable for all environments, also on staging. Now from line 69, you don't set it.

Reply:
@nesitor 
i don't see the problem here, the variable is set either in line 65 or in line 84
- in the label iteration, set the environment to production if 'deploy' label is found
- after iterating, since staging wins over production, set the environment to staging if 'deploy_staging' label was found.

...though i admit this logic may not be obvious to see.

Reply:
Yes, sorry, I don't seen it on line 84.